plugin/trino-geospatial/src/test/java/io/trino/plugin/geospatial/TestGeoFunctions.java

```diff
@@ -1346,31 +2036,63 @@ public void testSTInteriorRingN()
 
     private void assertInteriorRingN(String wkt, int index, String expected)
     {
-        assertFunction(format("ST_ASText(ST_InteriorRingN(ST_GeometryFromText('%s'), %d))", wkt, index), VARCHAR, expected);
+        assertThat(assertions.expression("ST_AsText(ST_InteriorRingN(geometry, index))")
```

Comment:
But these are not simple function calls, unfortunately, We'd need to change the essence of the test altogether (e.g., compare the geometry object directly instead of converting it to the text representation first). Exercise for a future PR.